lib/data/context.js

```diff
@@ -0,0 +1,50 @@
+const isGenerator = require('../utils').isGenerator;
+const DataTable = require('./table');
+
+module.exports = function (context) {
+
+  context.Data = function (dataTable) {
+    let data = detectDataType(dataTable);
+
+    return {
+      Scenario: function (title, opts, fn) {
+        if (typeof opts === 'function' && !fn) {
+          fn = opts;
+          opts = {};
+        }
+        data.forEach((dataRow) => {
+          opts.data = dataRow;
+          let test = context.Scenario(`${title} | ${dataRow}`, opts, fn);
+          test.inject.current = dataRow;
```

Comment:
Added this to documentation but removed check from container.

We need separate concerns, so container should no have hardcoded elements inside it which are related to the different parts of CodeceptJS
the only thing responsible for data management is located in `data/context`.